ArduCopter/Copter.cpp

```diff
@@ -144,6 +144,7 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
     SCHED_TASK(ekf_check,             10,     75),
     SCHED_TASK(gpsglitch_check,       10,     50),
     SCHED_TASK(landinggear_update,    10,     75),
+    SCHED_TASK(standby_update,        100,     75),
```

Comment:
nitpick: there's an extra space before the "75"

Reply:
Cool.